share/picongpu/dockerfiles/ubuntu-1604/Dockerfile

```diff
@@ -6,7 +6,7 @@ ENV        DEBIAN_FRONTEND=noninteractive \
            FORCE_UNSAFE_CONFIGURE=1 \
            SPACK_ROOT=/usr/local \
            SPACK_EXTRA_REPO=/usr/local/share/spack-repo \
-           PIC_PACKAGE='picongpu@0.5.0+isaac backend=cuda'
+           PIC_PACKAGE='picongpu@0.5.0-rc1+isaac backend=cuda'
```

Comment:
This tag here is the tag you provide in our [spack repo](https://github.com/ComputationalRadiationPhysics/spack-repo).

So add this before you can build.